sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items-mlflow.ipynb

```diff
@@ -0,0 +1,1162 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download run's artifacts/outputs\n",
+    "local_path = mlflow_client.download_artifacts(\n",
+    "    best_run.info.run_id, \"outputs\", local_dir\n",
+    ")\n",
+    "print(\"Artifacts downloaded in: {}\".format(local_path))\n",
+    "print(\"Artifacts: {}\".format(os.listdir(local_path)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "# Show the contents of the MLFlow model folder\n",
+    "os.listdir(\"./artifact_downloads/outputs/mlflow-model\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 6. Register best model and deploy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6.1 Create managed online endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import required libraries\n",
+    "from azure.ai.ml.entities import (\n",
+    "    ManagedOnlineEndpoint,\n",
+    "    ManagedOnlineDeployment,\n",
+    "    Model,\n",
+    "    Environment,\n",
+    "    CodeConfiguration,\n",
+    "    ProbeSettings,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Creating a unique endpoint name with current datetime to avoid conflicts\n",
+    "import datetime\n",
+    "\n",
+    "online_endpoint_name = \"ic-ml-fridge-items-\" + datetime.datetime.now().strftime(\n",
+    "    \"%m%d%H%M%f\"\n",
+    ")\n",
+    "\n",
+    "# create an online endpoint\n",
+    "endpoint = ManagedOnlineEndpoint(\n",
+    "    name=online_endpoint_name,\n",
+    "    description=\"this is a sample online endpoint for deploying model\",\n",
+    "    auth_mode=\"key\",\n",
+    "    tags={\"foo\": \"bar\"},\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.begin_create_or_update(endpoint)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6.2 Register best model and deploy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Register model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = \"ic-ml-fridge-items-model\"\n",
+    "model = Model(\n",
+    "    path=f\"azureml://jobs/{best_run.info.run_id}/outputs/artifacts/outputs/mlflow-model/\",\n",
+    "    name=model_name,\n",
+    "    description=\"my sample image classification multilabel model\",\n",
+    "    type=AssetTypes.MLFLOW_MODEL,\n",
+    ")\n",
+    "\n",
+    "# for downloaded file\n",
+    "# model = Model(path=\"artifact_downloads/outputs/model.pt\", name=model_name)\n",
```

Comment:
Please update this comment to have right model path pointing to downloaded mlflow-model directory. Also add type here. Please add it to all notebooks.